src/api/routes/v1/amendments.ts

```diff
@@ -54,7 +55,7 @@ interface AmendmentInVotingMap {
   }
 }
 
-interface AmendmentsEnabled extends AmendmentsInfo {
+interface AmendmentStatusExtended extends AmendmentsInfo {
```

Comment:
Based on the name, I would expect `AmendmentStatusExtended` to extend `AmendmentStatus`

(Also, if these are just optional parameters, would it make sense to include them as part of the original object as optional fields instead of having a separate type?)